src/content/docs/en/reference/api-reference.mdx

```diff
@@ -687,6 +687,35 @@ export function GET(context: APIContext) {
 }
 ```
 
+### `Astro.getActionResult()`
+
+`Astro.getActionResult()` is a function that returns the result of an [Action](/en/guides/actions/) submission. This accepts a an Action function as an argument (e.g. `actions.logout`), and returns a `data` or `error` object when a submission is received. Otherwise, it will return `undefined`.
+
+```astro title="src/pages/index.astro"
+---
+import { actions } from 'astro:actions';
+
+const result = Astro.getActionResult(actions.logout);
+---
+
+<form action={actions.logout}>
+  <button type="submit">Log out</button>
+</form>
+{result?.error && <p>Failed to log out. Please try again.</p>}
+```
+
+### `Astro.callAction()`
+
```

Comment:
The version seems correct (https://github.com/withastro/astro/pull/11590). However, I omitted the type here because of its complexity. I think it might be more confusing than helpful.

```suggestion

<p>

<Since v="4.13.1" />
</p>

```